plugins/core/Standard/qFacets/src/qFacets.cpp

```diff
@@ -1068,41 +1148,117 @@ void qFacets::exportFacetsInfo()
 	assert(!facets.empty());
 
 	FacetsExportDlg fDlg(FacetsExportDlg::ASCII_FILE_IO, m_app->getMainWindow());
+	
 	fDlg.orientationGroupBox->setEnabled(false);
+	
+	
+	
+	
+	
+	
+	
+	
 
 	//persistent settings (default export path)
 	QSettings settings;
 	settings.beginGroup("qFacets");
 	QString facetsSavePath = settings.value("exportPath", ccFileUtils::defaultDocPath()).toString();
+	
 	fDlg.destinationPathLineEdit->setText(facetsSavePath + QString("/facets.csv"));
 
 	if (!fDlg.exec())
 		return;
 
-	QString filename = fDlg.destinationPathLineEdit->text();
+	
 
+	QString filename = fDlg.destinationPathLineEdit->text();
 	//save current export path to persistent settings
 	settings.setValue("exportPath", QFileInfo(filename).absolutePath());
+	
+	bool coordsInCSV = fDlg.coordsInCSVCheckBox->isChecked();
+	
+	bool useNativeOrientation = fDlg.nativeOriRadioButton->isChecked();
+	bool useGlobalOrientation = fDlg.verticalOriRadioButton->isChecked();
+	bool useCustomOrientation = fDlg.customOriRadioButton->isChecked();
+	
+	double nX = 0.0f;
```

Comment:
```suggestion
	double nX = 0.0;
```

Same for the 2 next lines